papis/bibtex.py

```diff
@@ -301,7 +301,10 @@ def exporter(documents: List[papis.document.Document]) -> str:
 
 
 class Importer(papis.importer.Importer):
-    """Importer that parses BibTeX files."""
+    """
+    Importer that parses BibTeX files or strings.
+    Here, `uri` can either be a bibtex string, or point to a bibtex file or http/s url
```

Comment:
```suggestion
    Importer that parses BibTeX files or strings.
    
    Here, `uri` can either be a BibTeX string, local BibTeX file or a remote URL
    (with a HTTP or HTTPS protocol).
```
? I don't think that second part is displayed anywhere, but it's nice to have it capitalized :grin: